### ISSUE TITLE
Camera coordinate 

### ISSUE THREAD

**Issue by dogyoonlee** [google/nerfies#39]
Hello!

I really appreciate to your great work!!

However, I have some questions about camera coordinate on nerfies dataset  you released.

I want to use the NeRF code which is implemented on pytorch framework and it provide llff dataset dataloader.

As I know, llff dataset uses Normalized coordinate and camera poses which are saved as format of [-u, r, -t]

and they load the saved poses and rotate them as [r, u, -t] using follow codes.

**Comment by 2201855616** [comment]
The similar work I am doing. I also don't know how to transform blend dataset in Dnerf to nerfies dataset. 
And I also have read your issue in hypernerf.
(The answer may be in Nerfies Capture Processing v2.ipynb?)

**Comment by keunhong** [comment]
Hi Dogyoon.

1. We don't use NDC coordinates as NeRF does for some of the LLFF scenes. We use normal Euclidean coordinates, in which case the specific coordinate convention doesn't matter as long as it's consistent within the dataset. That being said, we use the [OpenCV convention](https://docs.opencv.org/4.x/dc/dbb/tutorial_py_calibration.html).
2. NSFF and the original NeRF paper pre-undistort the images with COLMAP. We instead do the undistortion on the fly with our camera class. This has some benefits, one of which is that if you pre-undistort you have to throw away some of the edge pixels.

If it's not training well there might be a bug in how the rays are generated in your code. I'd create a simple notebook and see if the generated rays are consistent between the JAX version and your PyTorch version.

You could also work around it by pre-undistorting the datasets with COLMAP and set the distortion to zero, but you'd have to write some code to load in the cameras from the other formats.
